compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -1030,6 +1041,20 @@ fn capture_upvar<'a, 'tcx>(
         captured_place_expr = Expr { temp_lifetime, ty: proj.ty, span: closure_expr.span, kind };
     }
 
+    captured_place_expr
+}
+
+fn capture_upvar<'a, 'tcx>(
```

Comment:
I probabbly should've have renamed it earlier, but maybe call this`convert_capture` since we capture Places and not just upvars.